folium/folium.py

```diff
@@ -664,6 +666,31 @@ def fit_bounds(self, bounds, padding_top_left=None,
         self.template_vars.update({'fit_bounds': fit_bounds_str})
 
 
+    def _auto_bounds(self):
+        if 'fit_bounds' in self.template_vars:
```

Comment:
Agreed.  Lets keep it simple then and open an issue with this idea so we do not get lost.